src/main/webapp/dev/js/common/feedbackResponseComments.es6

```diff
@@ -287,7 +322,8 @@ function showResponseCommentAddForm(recipientIndex, giverIndex, qnIndex, section
         });
         /* eslint-enable camelcase */
     }
-
+    saveInitialVisibilityOfCheckboxes(`showResponseCommentAddForm${id}`,
+        $(`#showResponseCommentAddForm${id}`).children('.responseCommentAddForm'));
```

Comment:
For future work: There is a lot of error-prone repetition here in the computation of `showResponseCommentAddForm${id}`. I'm letting this slide for now because there's already other cases of repetition in this and other functions, and I don't want this PR to have extraneous changes.